### DISCUSSION TITLE
Security - Notifications of Access + Retrieval of Auth History

### DISCUSSION THREAD

**Discussion by stevebissett** [Investec-Developer-Community/.github#18] [Ideas]
When it comes to security, there are currently no mechanisms that I'm aware of to: 
- Retrieve authentication/session history for a given API key/profile/account. 
- Notify on failed or successful auth via Investec App notifications. 
- Notify on failed or successful auth via REST webhooks. 

Notifications could be optional and turned on/off via the UI on profile. 

Scopes are useful for restricting access, but more functionality around access history and logins would be useful for security purposes and piece of mind.

**Comment by devinpearson** [comment]
This would be a fantastic addition to the API
